### PR TITLE
Toast Error on playground page

### LINKED ISSUE #6681: [FEATURE] Toast Error on playground page
## Solution

*update: Please see comment https://github.com/EddieHubCommunity/LinkFree/issues/6681#issuecomment-1541427897*

## Problem

On the [playground](https://linkfree.io/playground),
if we somehow forget to add github username or add invalid json, it shows an error on the top.
But it can be confusing for some users at first if they scroll all the way down to the bottom, meaning it doesn't indicate any error.

Here is a video to understand:

https://user-images.githubusercontent.com/77217706/236687975-e54e343b-d676-49f4-840a-3d53a3a507b7.mp4

## `The Feature`
If user forget to add something or anything that causes error. Put a toast rather than showing the button on top.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by AJcodes42**
## Fixes Issue

Closes #6681 

## Changes proposed

1) Moved Notification logic out of IconCard.js to the Notification.js
2) Replaced Alert component uses in playground.js with Notification component

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Behavior on Icons page remains same:
Screenshot 2023-05-17 at 4 30 29 PM

Error scenario in playground page show error notification:
Screenshot 2023-05-17 at 4 28 51 PM

Success scenario in playground page show success notification:
Screenshot 2023-05-17 at 4 29 26 PM

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6938"></a>

### CODE REVIEW

**Comment by eddiejaoude** [comment]
Thank you for your contribution 👍 The notification style should be consistent for all pages

File: components/Notification.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  let notificationAlignmentClass = "sm:items-center";
+  if (alignment === "left") {
+    notificationAlignmentClass = "sm:items-start";
+  } else if (alignment === "right") {
+    console.log("alignment right");
+    notificationAlignmentClass = "sm:items-end";
+  }
+
+  return (
+    <>
```
Comment (eddiejaoude) [inline components/Notification.js:30]: I don't think this wrapper is required?

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
The different position styles of the notification is not required - we can keep it simple for now

**Comment by AJcodes42** [comment]
Difference position styles removed from notification. Below are the screenshots

Screenshot 2023-05-18 at 10 24 44 AM
Screenshot 2023-05-18 at 10 24 31 AM

File: components/IconCard.js [hunk 2]
```diff
@@ -1,70 +1,33 @@
-import { Fragment, useState } from 'react'
-import { Transition } from '@headlessui/react'
-import { CheckCircleIcon } from '@heroicons/react/24/outline'
-import { XMarkIcon } from '@heroicons/react/20/solid'
+import { useState } from 'react'
```
Comment (eddiejaoude) [inline components/IconCard.js:1]: Not consistent coding style with the single quotes

File: components/IconCard.js [hunk 3]
```diff
@@ -1,70 +1,33 @@
...
-                  </div>
-                </div>
-              </div>
-            </div>
-          </Transition>
-        </div>
-      </div>
+      <Notification
+        show={showNotification}
+        type = "success"
```
Comment (eddiejaoude) [inline components/IconCard.js:25]: extra spacing here, VScode Prettier plugin will fix these formatting styles

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Looks good 👍  I left some inline comments (note I didn't mention all the coding style items, as they are the same)

**Comment by eddiejaoude** [comment]
It is so close to being merged, I will merged and reformat the files in a temporary branch

**Pull request merged**